### ISSUE TITLE
Create a set of labels to be used across all repos within the org

### ISSUE THREAD

**Issue by Rumyra** [mdn/mdn#285]
- [ ] What our our common labels (all lower case no hyphens)

  - [ ] `needs triage` Triage needed by staff and/or partners. Automatically applied when an issue is opened.
  - [ ] `good first issue` A good issue for newcomers to get started with.
  - [ ] `help wanted` If you know something about this, we would love your help!
  - [ ] `needs info` This needs more information to review or act on.
  - [ ] `expert help needed` This needs more information from a subject matter expert (SME).
  - [ ] `idle` Issues and pull requests with no activity for three months.
    - [ ] Update reusable workflow
  - [ ]  `on hold` Waiting on something else before this can be moved forward.
  - [ ]  `wontfix` Deemed to be outside the scope of the project, or it would require a significant amount of time and resources to fix. (trimmed: Deemed to be outside the scope of the project or would require significant time and resources to fix)
  - [ ]  `for later` Not planned at this time.
  - [ ]  `needs content update` Needs update to the content to support this change.
  - [ ]  `bug` Indicates an unexpected problem or unintended behavior.
  - [ ]  `chore` A routine task.
  - [ ]  `enhancement` Improves an existing feature.

#### Effort

- [ ] `effort: small` Task is a small effort.
- [ ] `effort: medium` Task is a medium effort.
- [ ] `effort: large` Task is large effort.

#### Priority

- [ ] `p0` Urgent. We will address this as soon as possible.
- [ ] `p1` We will address this soon and will provide capacity from our team for it in the next few releases.
- [ ] `p2` We want to address this but may have other higher priority items.
- [ ] `p3` We don't have visibility when this will be addressed.

```[tasklist]
### Tasks
- [ ] https://github.com/mdn/mdn/issues/290
- [x] https://github.com/mdn/mdn/issues/287
- [x] https://github.com/mdn/mdn/issues/286
- [x] https://github.com/mdn/mdn/issues/297
- [x] https://github.com/mdn/mdn/issues/299
- [x] https://github.com/mdn/mdn/issues/294
- [x] https://github.com/mdn/mdn/issues/298
- [x] https://github.com/mdn/mdn/issues/291
- [x] https://github.com/mdn/mdn/issues/295
- [x] https://github.com/mdn/mdn/issues/286
- [x] bob
- [x] https://github.com/mdn/mdn/issues/293
- [x] https://github.com/mdn/mdn/issues/292
- [x] https://github.com/mdn/mdn/issues/291
- [x] data
- [x] express-locallibrary-tutorial
- [x] workflows
- [x] todo-vue
- [x] todo-react
- [x] project-template
- [x] mdn-contributor-spotlight
- [x] html-examples
- [x] pwa-examples
- [x] release-dashboard
- [x] django-locallibrary-tutorial
- [x] beginner-html-site-styled
- [x] mdn-community
- [x] webaudio-examples
- [x] web-components-examples
- [x] beginner-html-site
- [x] django-diy-blog
- [x] webvr-tests
- [x] imsc-examples
- [x] infra
- [x] museum
- [x] js-examples
- [x] webassembly-examples
- [x] beginner-html-site-scripted
```

**Comment by dipikabh** [comment]
Labels to keep (non-hyphenated):
- needs triage
- expert help needed
- community help needed
- awaiting response
- awaiting review

**Comment by bsmth** [comment]
Most important:

- `good first issue`
- `help wanted` 
- `wontfix`
- `duplicate` -> are there any GitHub things to mark as duplicate (`duplicate of #123`?)

Personally, I wouldn't be sad if this goes away:

- `needs triage` this applies to all issues, I think this is redundant

**Comment by schalkneethling** [comment]
- good first issue
- help wanted
- chore
- bug
- needs triage
- p1-p3
- dependencies
- has dependency
- enhancement
- on hold

**Comment by schalkneethling** [comment]
> * -> are there any GitHub things to mark as duplicate (`duplicate of #123`?)

https://docs.github.com/en/issues/tracking-your-work-with-issues/marking-issues-or-pull-requests-as-a-duplicate

**Comment by Rumyra** [comment]
#### Time

- `time: -30 min`
- `time: -3 hr`
- `time: -2 day`

#### Priority

- `p0`
- `p1`
- `p2`
- `p3`

#### General

- `needs triage`
- `on hold`
- `good first issue`
- `help wanted`
- `idle`

#### Content specific which would be good on some other repos

- all `content ...`

**Comment by dipikabh** [comment]
Repos
- content
- yari
- interactive-examples

**Comment by schalkneethling** [comment]
Repos

- [ ] https://github.com/mdn/perf-examples
- [ ] https://github.com/mdn/todo-react-build
- [ ] https://github.com/mdn/pab
- [ ] https://github.com/mdn/web-tech-games
- [ ] https://github.com/mdn/houdini-examples
- [ ] https://github.com/mdn/differy
- [ ] https://github.com/mdn/beginner-html-site-scripted
- [ ] https://github.com/mdn/webassembly-examples
- [ ] https://github.com/mdn/js-examples
- [ ] https://github.com/mdn/museum
- [ ] https://github.com/mdn/infra
- [ ] https://github.com/mdn/imsc-examples
- [ ] https://github.com/mdn/webvr-tests
- [ ] https://github.com/mdn/django-diy-blog
- [ ] https://github.com/mdn/.github
- [ ] https://github.com/mdn/beginner-html-site
- [ ] https://github.com/mdn/web-components-examples
- [ ] https://github.com/mdn/webaudio-examples

**Comment by bsmth** [comment]
Looks like all of these are done now 🎉
